src/main/java/uk/gov/hmcts/reform/unspec/aspect/NoOngoingBusinessProcessAspect.java

```diff
@@ -37,7 +38,7 @@ public Object checkOngoingBusinessProcess(
         CaseEvent caseEvent = CaseEvent.valueOf(callbackParams.getRequest().getEventId());
         CaseDetails caseDetails = callbackParams.getRequest().getCaseDetails();
         CaseData caseData = caseDetailsConverter.toCaseData(caseDetails);
-        if (hasNoOngoingBusinessProcess(caseData)) {
+        if (hasNoOngoingBusinessProcess(caseData) || caseEvent.getUserType() == UserType.CAMUNDA) {
```

Comment:
commented in other PR, would be nice for this to exist as a method in caseEvent class